[debug] ----------------------------------------------------------------------
[debug] Command:      C:\Program Files\nodejs\node.exe C:\Users\ALI\AppData\Roaming\npm\node_modules\firebase-tools\bin\firebase deploy
[debug] CLI Version:  3.9.0
[debug] Platform:     win32
[debug] Node Version: v7.4.0
[debug] Time:         Tue Sep 26 2017 11:55:47 GMT-0700 (Pacific Daylight Time)
[debug] ----------------------------------------------------------------------
[debug] 
[debug] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] > refreshing access token with scopes: ["email","https://www.googleapis.com/auth/cloud-platform","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","openid"]
[debug] >>> HTTP REQUEST POST https://www.googleapis.com/oauth2/v3/token { refresh_token: '1/8YrbcfUQH2XheUikgwpidU_EctM3Ffc3FXXjTj1QhXM',
  client_id: '563584335869-fgrhgmd47bqnekij5i8b5pr03ho849e6.apps.googleusercontent.com',
  client_secret: 'j9iVZfS8kkCEFUPaAeJV0sAi',
  grant_type: 'refresh_token',
  scope: 'email https://www.googleapis.com/auth/cloud-platform https://www.googleapis.com/auth/cloudplatformprojects.readonly https://www.googleapis.com/auth/firebase openid' } 
 Tue Sep 26 2017 11:55:47 GMT-0700 (Pacific Daylight Time)
[debug] <<< HTTP RESPONSE 200
[debug] >>> HTTP REQUEST GET https://admin.firebase.com/v1/projects/meetup-d162c  
 Tue Sep 26 2017 11:55:48 GMT-0700 (Pacific Daylight Time)
[debug] <<< HTTP RESPONSE 200
[debug] >>> HTTP REQUEST GET https://admin.firebase.com/v1/database/meetup-d162c/tokens  
 Tue Sep 26 2017 11:55:49 GMT-0700 (Pacific Daylight Time)
[debug] <<< HTTP RESPONSE 200
[info] 
[info] === Deploying to 'meetup-d162c'...
[info] 
[debug] TypeError: targetNames.join is not a function
    at deploy (C:\Users\ALI\AppData\Roaming\npm\node_modules\firebase-tools\lib\deploy\index.js:68:57)
    at Command._action (C:\Users\ALI\AppData\Roaming\npm\node_modules\firebase-tools\commands\deploy.js:47:12)
    at C:\Users\ALI\AppData\Roaming\npm\node_modules\firebase-tools\lib\command.js:176:29
    at tryCatch (C:\Users\ALI\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\rsvp\dist\rsvp.js:539:12)
    at invokeCallback (C:\Users\ALI\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\rsvp\dist\rsvp.js:554:13)
    at publish (C:\Users\ALI\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\rsvp\dist\rsvp.js:522:7)
    at flush (C:\Users\ALI\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\rsvp\dist\rsvp.js:2414:5)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
    at process._tickCallback (internal/process/next_tick.js:98:9)
[error] 
[error] Error: An unexpected error has occurred.
